### ISSUE TITLE
FutureDirectivesExamplesSpec: 

### ISSUE THREAD

**Issue by raboof** [akka/akka-http#3311]
similar to #1419

https://jenkins.akka.io:8498/job/akka-http-nightly/AKKA_VERSION=release-2.5,Node=akka-http-nightly-node,SCALA_VERSION=2.12.11,jdk=AdoptOpenJDK%2011/178/console

```
[info] FutureDirectivesExamplesSpec:
[info] - onComplete
[info] - onCompleteWithBreaker *** FAILED ***
[info]   Request was rejected with rejection CircuitBreakerOpenRejection(akka.pattern.CircuitBreakerOpenException: Circuit Breaker is open; calls are failing fast) (FutureDirectivesExamplesSpec.scala:96)
```

**Comment by chbatey** [comment]
https://jenkins.akka.io:8498/job/akka-http-nightly/AKKA_VERSION=release-2.5,Node=akka-http-nightly-node,SCALA_VERSION=2.13.3,jdk=AdoptOpenJDK%208/317/testReport/junit/docs.http.scaladsl.server.directives/FutureDirectivesExamplesSpec/onCompleteWithBreaker/

**Comment by jrudolph** [comment]
Hopefully fixed in #3694.
